confluent_kafka/schema_registry/schema_registry_client.py

```diff
@@ -340,51 +444,42 @@ def delete_version(self, subject_name, version):
             https://docs.confluent.io/current/schema-registry/develop/api.html#delete--subjects-(string-%20subject)-versions-(versionId-%20version)
 
         """  # noqa: E501
-        self._rest_client.DELETE(url_formatter(VERSION,
-                                               subject_name,
+        return self._rest_client.DELETE('subjects/{}/versions/{}'.
+                                        format(_urlencode(subject_name),
                                                version))
 
     def set_compatibility(self, subject_name=None, level=None):
         """
-        PUT /config/{subject_name}
-
         Update global or subject level compatibility policy.
 
         Args:
-            - level (CompatibilityType): Compatibility policy
-
-        Keywork Args:
-            - subject_name (str, optional): Subject name.
+            level (CompatibilityType): Compatibility policy
```

Comment:
duplicate, and does not indicate that it is optional or what the defualt value is.